unbound_exporter.go

```diff
@@ -297,6 +304,15 @@ func CollectFromReader(file io.Reader, ch chan<- prometheus.Metric) error {
 				if err != nil {
 					return err
 				}
+
+				if !threads && strings.HasPrefix(matches[0], "thread") {
```

Comment:
Can you guarantee that matches actually has this length?

Reply:
I think I can. There is `matches != nil` check before ParseFloat(), so there should always be at least first entry. Assuming the first entry tells `thread` all of the expressions have three fields. That should make both the `matches[1]` assignment and `matches[1:]` reference safe.